### PR TITLE
Feature/sdb 8 registration summary

### PR DESCRIPTION
**Pull request by Djangofs**

### CODE REVIEW

File: features/step_definitions/stepdefs.js [hunk 1]
```diff
@@ -337,3 +337,46 @@ Then("my box is still ticked", async function() {
...
+    webdriver.By.name("establishment_trading_name")
+  );
+  await establishmentTradingName.sendKeys(validTradingName);
+  saveContinueButton = await driver.findElement(webdriver.By.className("css-nyvlzd"));
+  await saveContinueButton.submit();
+});
+
+Then("I am taken to the declaration page", async () => {
+  currentUrl = await driver.getCurrentUrl();
+  assert.notEqual(
```
Comment (Joe-Brady) [inline features/step_definitions/stepdefs.js]: Why are we checking for `notEqual`? (most likely just my lack of understanding)

File: features/step_definitions/stepdefs.js [hunk 2]
```diff
@@ -337,3 +337,46 @@ Then("my box is still ticked", async function() {
...
+  assert.equal(summaryTradingName, validTradingName, "Trading name does not match");
+});
+
+Then("The data I did not enter should not be displayed", async() => {
+  let summaryTradingName;
+  try {
+    summaryTradingName = await driver.findElement(
+      webdriver.By.id("operator_name")
+    );
+  } catch(err) {
```
Comment (Joe-Brady) [inline features/step_definitions/stepdefs.js:379]: Currently the `div` containing the text is actually still rendered, it just contains an empty string. So in theory the ID should still be found. Does that tie in with this approach?

Comment (Djangofs) [reply features/step_definitions/stepdefs.js:379]: hmm it was throwing an error and doing a manual check the div isn't there either. Would be preferable if it was rendering so I wouln't have to do the try / catch though

File: features/step_definitions/stepdefs.js (hunk 1 above)
Comment (Djangofs) [reply features/step_definitions/stepdefs.js]: Error on my part, fixing now

**Pull request merged**